src/lib.rs

```diff
@@ -81,7 +81,6 @@ pub use auto::*;
 
 pub mod prelude;
 
-#[allow(clippy::transmute_ptr_to_ref)]
```

Comment:
Probably needs the same change in glib then?

Reply:
Indeed!